wled00/udp.cpp

```diff
@@ -155,15 +160,11 @@ void notify(byte callMode, bool followUp)
     // send global data
     DEBUG_PRINTLN(F("ESP-NOW sending first packet."));
     const size_t bufferSize = sizeof(buffer.data)/sizeof(uint8_t);
-    size_t packetSize = 41;
-    size_t s0 = 0;
+    size_t packetSize = SEG_OFFSET;
     memcpy(buffer.data, udpOut, packetSize);
-    // stuff as many segments in first packet as possible (normally up to 5)
-    for (size_t i = 0; packetSize < bufferSize && i < s; i++) {
-      memcpy(buffer.data + packetSize, &udpOut[41+i*UDP_SEG_SIZE], UDP_SEG_SIZE);
-      packetSize += UDP_SEG_SIZE;
-      s0++;
-    }
+    // since we sync the name, only one segment can fit in the first packet
```

Comment:
This is a waste of packet space. Do not forget that the same packet is re-used in ESP-NOW sync.
ESP-NOW packet can only contain 250 bytes so "compression" of data is a must.